ding/envs/env_manager/envpool_env_manager.py

```diff
@@ -39,10 +54,12 @@ def default_config(cls) -> EasyDict:
         # Async mode: batch_size <  env_num
         env_num=8,
         batch_size=8,
+        image_observation=True,
     )
 
     def __init__(self, cfg: EasyDict) -> None:
-        self._cfg = cfg
+        self._cfg = self.default_config()
+        self._cfg.update(cfg)
```

Comment:
why merge config here, we have already merged the config of env manager in `compile_config` function

Reply:
There are two env manager config, one for evaluator and one for collector. It's too complicated to use compile_config with auto=True.
I suggest use  compile_config with auto=False.